pkg/views/workspace/selection/workspace.go

```diff
@@ -19,6 +19,13 @@ import (
 
 var NewWorkspaceIdentifier = "<NEW_WORKSPACE>"
 
+type ActionVerb string
+
+var StartActionVerb ActionVerb = "Start"
+var StopActionVerb ActionVerb = "Stop"
+var RestartActionVerb ActionVerb = "Restart"
+var DeleteActionVerb ActionVerb = "Delete"
+
 func generateWorkspaceList(workspaces []apiclient.WorkspaceDTO, isMultipleSelect bool, action string) []list.Item {
```

Comment:
We can make the `action` a `ActionVerb` so we can avoid casting to `string()` in a few places